packages/react-redux/modules/components/toggle-feature/toggle-feature.tsx

```diff
@@ -1,20 +1,19 @@
 import { TFlagName, TFlagVariation } from '@flopflip/types';
 
 import React from 'react';
-import { ToggleFeature as SharedToggleFeature } from '@flopflip/react';
+import {
+  ToggleFeature as SharedToggleFeature,
+  TToggleFeatureProps,
```

Comment:
The underlying props of the `ToggleFeature` component are now exported, so that we can "merge" them with the props from the "higher level" `ToggleFeature` components.
This should fix the issue with the "missing children", etc"